docs/design/fabric-v2+/fpc-management.md

```diff
@@ -0,0 +1,72 @@
+# FPC Management
+
+This document defines the management APIs of FPC.
+Such definition builds on the design diagrams describing the enclave creation, 
+the enclave registration and the chaincode key generation.
+
+## Admin Commands
+
+### Extended Fabric v2 Lifecycle Chaincode Commands
+
+These commands extend the original Fabric v2 commands to handle FPC chaincodes.
+For a description of the original Fabric v2 commands, check out the Fabric v2 documentation.
+In the following, this document describes additional flags and usages.
+Any other commands, which does not appear in this list, remains unchanged in FPC.
+
+#### Package
+
+This commands has a new option `--lang fpc-c` to specify the FPC chaincode type.
+
+#### Approveformyorg
+
+This command has the following requirements *when it is used for FPC chaincodes*:
+* `--version <base64-encoded MRENCLAVE>`, the version of the FPC chaincode must contain a string that represent the identity of the enclave. This is the same identity which the trusted hardware computes and attests to. The string can be conveniently found in the `mrenclave` output file of the `generate_mrenclave.sh` script.
+* `--signature-policy string`, the endorsement policy must be a valid FPC endorsement policy.
+FPC currently supports only a single enclave as endorser, running at a designated peer.
+* `--endorsement-plugin string`, this flag is not supported in FPC.
+* `--validation-plugin string`, this flag is not supported in FPC.
+
+#### Checkcommitreadiness
+
+This command has the following requirements *when it is used for FPC chaincodes*:
+* `--version <base64-encoded MRENCLAVE>`, the version of the FPC chaincode must contain a string that represent the identity of the enclave. This is the same identity which the trusted hardware computes and attests to. The string can be conveniently found in the `mrenclave` output file of the `generate_mrenclave.sh` script.
+* `--signature-policy string`, the endorsement policy must be a valid FPC endorsement policy.
+FPC currently supports only a single enclave as endorser, running at a designated peer.
+* `--endorsement-plugin string`, this flag is not supported in FPC.
+* `--validation-plugin string`, this flag is not supported in FPC.
```

Comment:
From a maintainance perspective easier 
Maybe just to refer to approveformyorg section rather than duplicating it? Will make our life easier when updating but also i think makes it explicit that exactly the same reasoning applies in all three commands
```suggestion
This command has the same requirements *when it is used for FPC chaincodes* as described above for `approveformyorg`.
```